### DISCUSSION TITLE
real deletion of all transactions

### DISCUSSION THREAD

**Discussion by timaschew** [firefly-iii/firefly-iii#6398] [Firefly III] [answered]
On the `/profile` page I've found a button to delete all transactions. But they are still in the database because I can't import them again. That is mentioned in the docs, but the docs tells also that I can turn the duplicatation detection off, but that didn't work as I've described in https://github.com/orgs/firefly-iii/discussions/6353#discussioncomment-3525598

So my question is if it's possible to add to the UI a checkbox if you want to delete the transactions completely from the database.

**Comment by JC5** [comment, marked as answer]
Yes, this is on my list for the new layout :+1:
